### PR TITLE
prepare readme to release

### PR DESCRIPTION
**Pull request by dshil**
A small typo fix in a section describing how to install gits.

### CODE REVIEW

**Comment by dshil** [comment]
ping @eugulixes, I hope to release gits on next Monday, for achieving this goal I want to slightly improve our docs.

During a building gits for varios platforms I've reallised that living with a python and nodejs at the same time is a big pain in the ass.

Let's begin with Linux Mint:
- It doesn't have a setuptools on a startup.
- nodejs has a different location, that's why we need to create a symlink to make our `npm run start` works.

Fedora 24:
- A zero problem with an installation (of course it's a good point).
- Broken htop, vim, nano, man (I wan't provide a screenshot demonstrating it, but all console oriented tools work incorrectly)
- Also I have a short dance with a tambourine with installing pip, virtualenv properly

MacOS:
- SSH deamon is always turn off on a start up and I always forget to turn it on and spend some time to figure it out WTF with our login util, that's why I think we need to mention that SSH deamon should be start before running gits.

I specify only a little problems above, because I didn't notice all problems and forgot some of them. 


Overall wishes:
- We need to rework CI by adding different platforms (#95).
- We need to provide a more simple interface for running gits (I think we should have something like `gits start` and that's it, user doesn't care about running a client at first, that running a server), we can achieve this by providing a tool, that will operate our low-level python lib and npm package.
- We need to provide a make file with set of simple targets: build, test. By build target I mean that user will have a ready tool to run gits on any platform (execpt windows). By test target I mean a simple way of running all gits test (now we don't have it).
- Can we move our doc back to github from readthedoc, it's a personal experience, but it's more convinient to have all stuff about gits in a one place.

**Comment by eugulixes** [comment]
@dshil, `gits start` would be a good idea.
You can try to amend and return our documentation to GiitHub.

File: docs/index.rst [hunk 1]
```diff
@@ -86,6 +86,10 @@ used through the ``--static-path`` and ``--templates-path`` parameters.
 
 Gits uses
 
+* `SSH <https://en.wikipedia.org/wiki/Secure_Shell>`_ to remotely login into a 
+  system. You need to ensure, that a ssh demon is running, before you'll start
```
Comment (eugulixes) [inline docs/index.rst]: Demon and daemon mean the same in religious context, but daemon is more common in Unix, according to my humble experience.

Comment (dshil) [reply docs/index.rst]: Ok, fixed.

Comment (dshil) [reply docs/index.rst]: @eugulixes, I hope to merge it today, to ensure, that tomorrow it'll be released.

**Review by eugulixes — APPROVED** [review]

**Pull request merged**